python/ray/cloudpickle/cloudpickle.py

```diff
@@ -59,6 +58,14 @@
 import uuid
 import threading
 
+PICKLE5_ENABLED = False
```

Comment:
This should only be needed for cloudpickle_fast, right? Let's focus on supporting that codepath, so it doesn't break other things.

Reply:
If we we have pickle5 enabled (python3 && python<3.8 && pickle5 installed), then we will try to reuse the old cloudpickle